scripts/validation/validate-pr-template-structure.cjs

```
@@ -0,0 +1,181 @@
+    'Checklist',
+  ];
+
+  for (const section of requiredSections) {
+    const sectionRegex = new RegExp(`^##\\s+${section}`, 'm');
+    if (!sectionRegex.test(templateContent)) {
+      errors.push(`Missing required section: "${section}"`);
+    }
+  }
+
+  // Changelog section should have subsections
+  const changelogSectionRegex = /^##\s+Changelog\s*$[\s\S]*?(?=^##\s|\Z)/m;
```

ESLint

[error] Unnecessary escape character: \Z.

(no-useless-escape)